Build failure: jack1

## Steps To Reproduce

Steps to reproduce the behavior:

1. build _jack1_ on current nixpkgs-unstable (de1864217bfa9b5845f465e771e0ecb48b30e02d)


## Build log

<details>



<summary>Build Log</summary>

Build step onwards, using `nix log /nix/store/g3dqklg07bimzb4xvbb9klhk0bnygmml-jack1-0.125.0.drv | sed '0,/^Running phase: buildPhase$/d;s/\x1B\[[0-9;]*[a-zA-Z]//g' | xclip -selection clipboard`:

```console
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: SHELL=/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/bash
make  all-recursive
make[1]: Entering directory '/build/jack-audio-connection-kit-0.125.0'
Making all in libjack
make[2]: Entering directory '/build/jack-audio-connection-kit-0.125.0/libjack'
  CC       libjackcommon_la-client.lo
client.c: In function 'server_event_connect':
client.c:839:25: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  839 |         jack_uuid_copy (&req.client_id, client->control->uuid);
      |                         ^~~~~~~~~~~~~~
client.c: In function 'jack_request_client':
client.c:1096:53: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 1096 |                 if (jack_uuid_parse (va->sess_uuid, &req.uuid) != 0) {
      |                                                     ^~~~~~~~~
client.c:1101:34: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 1101 |                 jack_uuid_clear (&req.uuid);
      |                                  ^~~~~~~~~
client.c: In function 'jack_set_freewheel':
client.c:1558:25: warning: taking address of packed member of 'union <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 1558 |         jack_uuid_copy (&request.x.client_id, client->control->uuid);
      |                         ^~~~~~~~~~~~~~~~~~~~
client.c: In function 'jack_session_reply':
client.c:1584:33: warning: taking address of packed member of 'union <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 1584 |                 jack_uuid_copy (&request.x.client_id, client->control->uuid);
      |                                 ^~~~~~~~~~~~~~~~~~~~
client.c: In function 'jack_activate':
client.c:2467:25: warning: taking address of packed member of 'union <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 2467 |         jack_uuid_copy (&req.x.client_id, client->control->uuid);
      |                         ^~~~~~~~~~~~~~~~
client.c: In function 'jack_deactivate_aux':
client.c:2483:41: warning: taking address of packed member of 'union <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 2483 |                         jack_uuid_copy (&req.x.client_id, client->control->uuid);
      |                                         ^~~~~~~~~~~~~~~~
client.c: In function 'jack_get_client_name_by_uuid':
client.c:2917:40: warning: taking address of packed member of 'union <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 2917 |         if (jack_uuid_parse (uuid_str, &request.x.client_id) != 0) {
      |                                        ^~~~~~~~~~~~~~~~~~~~
client.c: In function 'jack_reserve_client_name':
client.c:2971:40: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 2971 |         if (jack_uuid_parse (uuid_str, &request.x.reservename.uuid) != 0) {
      |                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'jack_handle_reorder',
    inlined from 'jack_client_process_events' at client.c:1876:13:
client.c:745:24: warning: '%u' directive writing between 1 and 10 bytes into a region of size between 0 and 4096 [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wformat-overflow�-Wformat-overflow=�]8;;�]
  745 |         sprintf (path, "%s-%" PRIu32, client->fifo_prefix, event->x.n);
      |                        ^~~~~~
client.c: In function 'jack_client_process_events':
client.c:745:28: note: format string is defined here
  745 |         sprintf (path, "%s-%" PRIu32, client->fifo_prefix, event->x.n);
In file included from /nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/stdio.h:970,
                 from client.c:28:
In function 'sprintf',
    inlined from 'jack_handle_reorder' at client.c:745:2,
    inlined from 'jack_client_process_events' at client.c:1876:13:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/bits/stdio2.h:30:10: note: '__builtin___sprintf_chk' output between 3 and 4108 bytes into a destination of size 4097
   30 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   31 |                                   __glibc_objsize (__s), __fmt,
      |                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   32 |                                   __va_arg_pack ());
      |                                   ~~~~~~~~~~~~~~~~~
In function 'jack_handle_reorder',
    inlined from 'jack_client_process_events' at client.c:1876:13:
client.c:754:24: warning: '%u' directive writing between 1 and 10 bytes into a region of size between 0 and 4096 [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wformat-overflow�-Wformat-overflow=�]8;;�]
  754 |         sprintf (path, "%s-%" PRIu32, client->fifo_prefix, event->x.n + 1);
      |                        ^~~~~~
client.c: In function 'jack_client_process_events':
client.c:754:28: note: format string is defined here
  754 |         sprintf (path, "%s-%" PRIu32, client->fifo_prefix, event->x.n + 1);
In function 'sprintf',
    inlined from 'jack_handle_reorder' at client.c:754:2,
    inlined from 'jack_client_process_events' at client.c:1876:13:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/bits/stdio2.h:30:10: note: '__builtin___sprintf_chk' output between 3 and 4108 bytes into a destination of size 4097
   30 |   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   31 |                                   __glibc_objsize (__s), __fmt,
      |                                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   32 |                                   __va_arg_pack ());
      |                                   ~~~~~~~~~~~~~~~~~
  CC       libjackcommon_la-intclient.lo
intclient.c: In function 'jack_get_internal_client_name':
intclient.c:105:25: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  105 |         jack_uuid_copy (&req.x.intclient.uuid, intclient);
      |                         ^~~~~~~~~~~~~~~~~~~~~
intclient.c: In function 'jack_internal_client_unload':
intclient.c:206:33: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  206 |                 jack_uuid_copy (&req.x.intclient.uuid, intclient);
      |                                 ^~~~~~~~~~~~~~~~~~~~~
intclient.c: In function 'jack_intclient_request.constprop.isra':
intclient.c:75:17: warning: '__builtin_strncpy' specified bound 4097 equals destination size [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-stringop-truncation�-Wstringop-truncation�]8;;�]
   75 |                 strncpy (req.x.intclient.path, va->load_name,
      |                 ^
intclient.c: In function 'jack_internal_client_handle':
intclient.c:138:9: warning: '__builtin_strncpy' specified bound 33 equals destination size [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-stringop-truncation�-Wstringop-truncation�]8;;�]
  138 |         strncpy (req.x.intclient.name, client_name,
      |         ^
  CC       libjackcommon_la-messagebuffer.lo
  CC       libjackcommon_la-metadata.lo
metadata.c: In function 'jack_property_change_notify':
metadata.c:128:25: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  128 |         jack_uuid_copy (&req.x.property.uuid, uuid);
      |                         ^~~~~~~~~~~~~~~~~~~~
  CC       libjackcommon_la-midiport.lo
  CC       libjackcommon_la-pool.lo
  CC       libjackcommon_la-port.lo
port.c: In function 'jack_port_register':
port.c:273:25: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  273 |         jack_uuid_copy (&req.x.port_info.client_id, client->control->uuid);
      |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~
port.c: In function 'jack_port_unregister':
port.c:301:25: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  301 |         jack_uuid_copy (&req.x.port_info.client_id, client->control->uuid);
      |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~
port.c: In function 'jack_port_get_all_connections':
port.c:404:26: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  404 |         jack_uuid_clear (&req.x.port_info.client_id);
      |                          ^~~~~~~~~~~~~~~~~~~~~~~~~~
port.c: In function 'jack_port_rename':
port.c:821:9: warning: 'jack_port_set_name' is deprecated [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations�-Wdeprecated-declarations�]8;;�]
  821 |         if ((ret = jack_port_set_name (port, new_name)) == 0) {
      |         ^~
In file included from port.c:28:
../jack/jack.h:766:5: note: declared here
  766 | int jack_port_set_name (jack_port_t *port, const char *port_name) JACK_OPTIONAL_WEAK_DEPRECATED_EXPORT;
      |     ^~~~~~~~~~~~~~~~~~
  CC       libjackcommon_la-ringbuffer.lo
  CC       libjackcommon_la-shm.lo
  CC       libjackcommon_la-thread.lo
  CC       libjackcommon_la-time.lo
  CC       libjackcommon_la-transclient.lo
transclient.c: In function 'jack_generate_unique_id':
transclient.c:42:34: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
   42 |         return exchange_and_add (&ectl->seq_number, 1);
      |                                  ^~~~~~~~~~~~~~~~~
transclient.c: In function 'jack_release_timebase':
transclient.c:365:25: warning: taking address of packed member of 'union <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  365 |         jack_uuid_copy (&req.x.client_id, ctl->uuid);
      |                         ^~~~~~~~~~~~~~~~
transclient.c: In function 'jack_set_sync_callback':
transclient.c:392:25: warning: taking address of packed member of 'union <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  392 |         jack_uuid_copy (&req.x.client_id, ctl->uuid);
      |                         ^~~~~~~~~~~~~~~~
transclient.c: In function 'jack_set_timebase_callback':
transclient.c:427:25: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  427 |         jack_uuid_copy (&req.x.timebase.client_id, ctl->uuid);
      |                         ^~~~~~~~~~~~~~~~~~~~~~~~~
  CC       libjackcommon_la-unlock.lo
unlock.c: In function 'cleanup_mlock':
unlock.c:86:17: warning: ignoring return value of 'fscanf' declared with attribute 'warn_unused_result' [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-unused-result�-Wunused-result�]8;;�]
   86 |                 fscanf (map, " %[^\n]", path);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  CC       libjackcommon_la-uuid.lo
  CCLD     libjackcommon.la
ar: `u' modifier ignored since `D' is the default (see `U')
/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/bash ../libtool --mode=compile gcc -I.. -I../config -I.. -I../include -I../include -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -Wall   -c -o simd.lo ./simd.c
libtool: compile:  gcc -I.. -I../config -I.. -I../include -I../include -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -Wall -c ./simd.c  -fPIC -DPIC -o .libs/simd.o
  CCLD     libjack.la
  CC       libjackdaemon_la-driver.lo
  CC       libjackdaemon_la-systemtest.lo
  CC       libjackdaemon_la-sanitycheck.lo
  CCLD     libjackdaemon.la
ar: `u' modifier ignored since `D' is the default (see `U')
make[2]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/libjack'
Making all in jackd
make[2]: Entering directory '/build/jack-audio-connection-kit-0.125.0/jackd'
  CC       jackd.o
In file included from jackd.c:44:
../include/driver_parse.h: In function 'jack_print_driver_options':
../include/driver_parse.h:45:29: warning: the comparison will always evaluate as 'true' for the address of 'str' will never be NULL [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress�-Waddress�]8;;�]
   45 |                         if (desc->params[i].value.str &&
      |                             ^~~~
In file included from ../include/engine.h:26,
                 from jackd.c:40:
../include/driver_interface.h:56:14: note: 'str' declared here
   56 |         char str[JACK_DRIVER_PARAM_STRING_MAX + 1];
      |              ^~~
jackd.c: In function 'do_nothing_handler':
jackd.c:92:9: warning: ignoring return value of 'write' declared with attribute 'warn_unused_result' [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-unused-result�-Wunused-result�]8;;�]
   92 |         write (1, buf, strlen (buf));
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
jackd.c: In function 'main':
jackd.c:215:17: warning: '__builtin_strncpy' specified bound 33 equals destination size [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-stringop-truncation�-Wstringop-truncation�]8;;�]
  215 |                 strncpy (req.x.intclient.name, client_name, sizeof(req.x.intclient.name));
      |                 ^
jackd.c:216:17: warning: '__builtin_strncpy' specified bound 4097 equals destination size [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-stringop-truncation�-Wstringop-truncation�]8;;�]
  216 |                 strncpy (req.x.intclient.path, path, sizeof(req.x.intclient.path));
      |                 ^
jackd.c:219:25: warning: '__builtin_strncpy' specified bound 1024 equals destination size [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-stringop-truncation�-Wstringop-truncation�]8;;�]
  219 |                         strncpy (req.x.intclient.init, args, sizeof(req.x.intclient.init));
      |                         ^
jackd.c:215:17: warning: '__builtin_strncpy' specified bound 33 equals destination size [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-stringop-truncation�-Wstringop-truncation�]8;;�]
  215 |                 strncpy (req.x.intclient.name, client_name, sizeof(req.x.intclient.name));
      |                 ^
jackd.c:216:17: warning: '__builtin_strncpy' specified bound 4097 equals destination size [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-stringop-truncation�-Wstringop-truncation�]8;;�]
  216 |                 strncpy (req.x.intclient.path, path, sizeof(req.x.intclient.path));
      |                 ^
  CC       libjackserver_la-engine.lo
engine.c: In function 'jack_do_get_uuid_by_client_name':
engine.c:2679:41: warning: taking address of packed member of 'union <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 2679 |                         jack_uuid_copy (&req->x.client_id, engine->driver->internal_client->control->uuid);
      |                                         ^~~~~~~~~~~~~~~~~
engine.c:2687:44: warning: passing argument 1 of 'strcmp' discards 'volatile' qualifier from pointer target type [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdiscarded-qualifiers�-Wdiscarded-qualifiers�]8;;�]
 2687 |                 if (strcmp (client->control->name, req->x.name) == 0) {
      |                             ~~~~~~~~~~~~~~~^~~~~~
In file included from /nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/sys/un.h:38,
                 from engine.c:27:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/string.h:156:32: note: expected 'const char *' but argument is of type 'volatile char *'
  156 | extern int strcmp (const char *__s1, const char *__s2)
      |                    ~~~~~~~~~~~~^~~~
engine.c:2688:41: warning: taking address of packed member of 'union <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 2688 |                         jack_uuid_copy (&req->x.client_id, client->control->uuid);
      |                                         ^~~~~~~~~~~~~~~~~
engine.c: In function 'jack_deliver_event':
engine.c:2980:58: warning: passing argument 2 of 'jack_client_handle_port_connection' discards 'const' qualifier from pointer target type [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdiscarded-qualifiers�-Wdiscarded-qualifiers�]8;;�]
 2980 |                                 (client->private_client, event);
      |                                                          ^~~~~
In file included from engine.c:46:
../include/internal.h:503:62: note: expected 'jack_event_t *' but argument is of type 'const jack_event_t *'
  503 |                                                jack_event_t *event);
      |                                                ~~~~~~~~~~~~~~^~~~~
engine.c:3025:86: warning: passing argument 2 of 'jack_client_handle_latency_callback' discards 'const' qualifier from pointer target type [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdiscarded-qualifiers�-Wdiscarded-qualifiers�]8;;�]
 3025 |                         jack_client_handle_latency_callback (client->private_client, event, (client->control->type == ClientDriver));
      |                                                                                      ^~~~~
../include/internal.h:549:85: note: expected 'jack_event_t *' but argument is of type 'const jack_event_t *'
  549 | extern int jack_client_handle_latency_callback(jack_client_t *client, jack_event_t *event, int is_driver);
      |                                                                       ~~~~~~~~~~~~~~^~~~~
engine.c: In function 'jack_port_do_register':
engine.c:4450:25: warning: taking address of packed member of 'struct _jack_port_shared' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 4450 |         jack_uuid_copy (&shared->client_id, req->x.port_info.client_id);
      |                         ^~~~~~~~~~~~~~~~~~
engine.c: In function 'jack_property_change_notify':
engine.c:4748:25: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 4748 |         jack_uuid_copy (&event.x.uuid, uuid);
      |                         ^~~~~~~~~~~~~
engine.c: In function 'jack_wake_server_thread':
engine.c:4867:9: warning: ignoring return value of 'write' declared with attribute 'warn_unused_result' [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-unused-result�-Wunused-result�]8;;�]
 4867 |         write (engine->cleanup_fifo[1], &c, 1);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
engine.c: In function 'jack_get_fifo_fd':
engine.c:4160:43: warning: '%d' directive output may be truncated writing between 1 and 11 bytes into a region of size between 0 and 4096 [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wformat-truncation�-Wformat-truncation=�]8;;�]
 4160 |         snprintf (path, sizeof(path), "%s-%d", engine->fifo_prefix,
      |                                           ^~
engine.c:4160:39: note: using the range [-2147483648, 2147483647] for directive argument
 4160 |         snprintf (path, sizeof(path), "%s-%d", engine->fifo_prefix,
      |                                       ^~~~~~~
In file included from /nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/stdio.h:970,
                 from engine.c:31:
In function 'snprintf',
    inlined from 'jack_get_fifo_fd' at engine.c:4160:2:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/bits/stdio2.h:68:10: note: '__builtin___snprintf_chk' output between 3 and 4109 bytes into a destination of size 4097
   68 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   69 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   70 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
engine.c: In function 'jack_do_session_notify':
engine.c:2806:83: warning: '%s' directive output may be truncated writing up to 32 bytes into a region of size between 0 and 255 [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wformat-truncation�-Wformat-truncation=�]8;;�]
 2806 |                                 snprintf (event.x.name, sizeof(event.x.name), "%s/%s/", req->x.session.path, client->control->name );
      |                                                                                   ^~
In function 'snprintf',
    inlined from 'jack_do_session_notify' at engine.c:2806:5:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/bits/stdio2.h:68:10: note: '__builtin___snprintf_chk' output between 3 and 290 bytes into a destination of size 256
   68 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   69 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   70 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
engine.c: In function 'jack_do_session_notify':
engine.c:2804:82: warning: '%s' directive output may be truncated writing up to 32 bytes into a region of size between 1 and 256 [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wformat-truncation�-Wformat-truncation=�]8;;�]
 2804 |                                 snprintf (event.x.name, sizeof(event.x.name), "%s%s/", req->x.session.path, client->control->name );
      |                                                                                  ^~
In function 'snprintf',
    inlined from 'jack_do_session_notify' at engine.c:2804:5:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/bits/stdio2.h:68:10: note: '__builtin___snprintf_chk' output between 2 and 289 bytes into a destination of size 256
   68 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   69 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   70 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
  CC       libjackserver_la-clientengine.lo
clientengine.c: In function 'jack_setup_client_control':
clientengine.c:622:33: warning: passing argument 1 of 'jack_uuid_copy' discards 'volatile' qualifier from pointer target type [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdiscarded-qualifiers�-Wdiscarded-qualifiers�]8;;�]
  622 |                 jack_uuid_copy (&client->control->uuid, uuid);
      |                                 ^~~~~~~~~~~~~~~~~~~~~~
In file included from clientengine.c:43:
../jack/uuid.h:39:42: note: expected 'jack_uuid_t *' {aka 'long unsigned int *'} but argument is of type 'volatile jack_uuid_t *' {aka 'volatile long unsigned int *'}
   39 | extern void jack_uuid_copy (jack_uuid_t* dst, jack_uuid_t src);
      |                             ~~~~~~~~~~~~~^~~
clientengine.c:622:33: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  622 |                 jack_uuid_copy (&client->control->uuid, uuid);
      |                                 ^~~~~~~~~~~~~~~~~~~~~~
clientengine.c: In function 'jack_client_create':
clientengine.c:925:45: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
  925 |                                req.options, &res.status, client_fd,
      |                                             ^~~~~~~~~~~
clientengine.c: In function 'jack_intclient_handle_request':
clientengine.c:1138:33: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 1138 |                 jack_uuid_copy (&req->x.intclient.uuid, client->control->uuid);
      |                                 ^~~~~~~~~~~~~~~~~~~~~~
clientengine.c: In function 'jack_intclient_load_request':
clientengine.c:1165:34: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 1165 |                 jack_uuid_clear (&req->x.intclient.uuid);
      |                                  ^~~~~~~~~~~~~~~~~~~~~~
clientengine.c:1168:33: warning: taking address of packed member of 'struct <anonymous>' may result in an unaligned pointer value [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Waddress-of-packed-member�-Waddress-of-packed-member�]8;;�]
 1168 |                 jack_uuid_copy (&req->x.intclient.uuid, client->control->uuid);
      |                                 ^~~~~~~~~~~~~~~~~~~~~~
clientengine.c: In function 'jack_get_reserved_name':
clientengine.c:861:33: warning: pointer 'reservation' used after 'free' [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-use-after-free�-Wuse-after-free�]8;;�]
  861 |                                 jack_slist_remove (engine->reserved_client_names, reservation);
      |                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
clientengine.c:859:25: note: call to 'free' here
  859 |                         free (reservation);
      |                         ^~~~~~~~~~~~~~~~~~
  CC       libjackserver_la-transengine.lo
  CC       libjackserver_la-controlapi.lo
In function 'get_realtime_priority_constraint',
    inlined from 'jackctl_server_create' at controlapi.c:897:6:
controlapi.c:839:31: warning: array subscript 'jack_driver_param_constraint_desc_t[0]' is partly outside array bounds of 'unsigned char[128]' [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Warray-bounds�-Warray-bounds=�]8;;�]
  839 |         constraint_ptr->flags = JACK_CONSTRAINT_FLAG_RANGE;
controlapi.c:834:64: note: object of size 128 allocated by 'calloc'
  834 |         constraint_ptr = (jack_driver_param_constraint_desc_t*)calloc (1, sizeof(jack_driver_param_value_enum_t));
      |                                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
controlapi.c:841:48: warning: array subscript 'jack_driver_param_constraint_desc_t[0]' is partly outside array bounds of 'unsigned char[128]' [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Warray-bounds�-Warray-bounds=�]8;;�]
  841 |         constraint_ptr->constraint.range.min.i = min;
      |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~
controlapi.c:834:64: note: object of size 128 allocated by 'calloc'
  834 |         constraint_ptr = (jack_driver_param_constraint_desc_t*)calloc (1, sizeof(jack_driver_param_value_enum_t));
      |                                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
controlapi.c:842:48: warning: array subscript 'jack_driver_param_constraint_desc_t[0]' is partly outside array bounds of 'unsigned char[128]' [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Warray-bounds�-Warray-bounds=�]8;;�]
  842 |         constraint_ptr->constraint.range.max.i = max;
      |         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~
controlapi.c:834:64: note: object of size 128 allocated by 'calloc'
  834 |         constraint_ptr = (jack_driver_param_constraint_desc_t*)calloc (1, sizeof(jack_driver_param_value_enum_t));
      |                                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  CCLD     libjackserver.la
  CCLD     jackd
echo "#define JACKD_MD5_SUM \"`md5 -q .libs/jackd | awk '{print $1}'`\"" > jack_md5.h
/nix/store/5mh7kaj2fyv8mk4sfq1brwxgc02884wi-bash-5.2p37/bin/bash: line 1: md5: command not found
make  all-am
make[3]: Entering directory '/build/jack-audio-connection-kit-0.125.0/jackd'
make[3]: Nothing to be done for 'all-am'.
make[3]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/jackd'
make[2]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/jackd'
Making all in drivers
make[2]: Entering directory '/build/jack-audio-connection-kit-0.125.0/drivers'
Making all in alsa_midi
make[3]: Entering directory '/build/jack-audio-connection-kit-0.125.0/drivers/alsa_midi'
  CC       alsa_midi.lo
  CC       port.lo
  CC       port_hash.lo
  CC       port_thread.lo
  CC       list.lo
  CCLD     jack_alsa_midi.la
make[3]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/drivers/alsa_midi'
Making all in alsa
make[3]: Entering directory '/build/jack-audio-connection-kit-0.125.0/drivers/alsa'
  CC       memops.lo
  CCLD     libmemops.la
ar: `u' modifier ignored since `D' is the default (see `U')
  CC       alsa_driver.lo
  CC       generic_hw.lo
  CC       hammerfall.lo
  CC       hdsp.lo
hdsp.c:42:18: warning: 'hdsp_audio_stream_index' defined but not used [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-unused-const-variable�-Wunused-const-variable=�]8;;�]
   42 | static const int hdsp_audio_stream_index[] = {
      |                  ^~~~~~~~~~~~~~~~~~~~~~~
  CC       ice1712.lo
  CC       usx2y.lo
  CCLD     jack_alsa.la
make[3]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/drivers/alsa'
Making all in dummy
make[3]: Entering directory '/build/jack-audio-connection-kit-0.125.0/drivers/dummy'
  CC       dummy_driver.lo
  CCLD     jack_dummy.la
make[3]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/drivers/dummy'
Making all in oss
make[3]: Entering directory '/build/jack-audio-connection-kit-0.125.0/drivers/oss'
  CC       oss_driver.lo
oss_driver.c: In function 'io_thread':
oss_driver.c:945:25: warning: ignoring return value of 'write' declared with attribute 'warn_unused_result' [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wno-unused-result�-Wunused-result�]8;;�]
  945 |                         write (driver->outfd, localbuf, localsize);
      |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  CCLD     jack_oss.la
make[3]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/drivers/oss'
Making all in firewire
make[3]: Entering directory '/build/jack-audio-connection-kit-0.125.0/drivers/firewire'
  CC       ffado_driver.lo
ffado_driver.c: In function 'ffado_driver_attach':
ffado_driver.c:186:50: warning: '%s' directive output may be truncated writing up to 63 bytes into a region of size between 51 and 61 [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wformat-truncation�-Wformat-truncation=�]8;;�]
  186 |                         snprintf (buf2, 64, "C%d_%s", (int)chn, buf); // needed to avoid duplicate names
      |                                                  ^~             ~~~
In file included from /nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/stdio.h:970,
                 from ffado_driver.c:33:
In function 'snprintf',
    inlined from 'ffado_driver_attach' at ffado_driver.c:186:4:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/bits/stdio2.h:68:10: note: '__builtin___snprintf_chk' output between 4 and 77 bytes into a destination of size 64
   68 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   69 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   70 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
ffado_driver.c: In function 'ffado_driver_attach':
ffado_driver.c:168:50: warning: '%s' directive output may be truncated writing up to 63 bytes into a region of size between 51 and 61 [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wformat-truncation�-Wformat-truncation=�]8;;�]
  168 |                         snprintf (buf2, 64, "C%d_%s", (int)chn, buf); // needed to avoid duplicate names
      |                                                  ^~             ~~~
In function 'snprintf',
    inlined from 'ffado_driver_attach' at ffado_driver.c:168:4:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/bits/stdio2.h:68:10: note: '__builtin___snprintf_chk' output between 4 and 77 bytes into a destination of size 64
   68 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   69 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   70 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
ffado_driver.c: In function 'ffado_driver_attach':
ffado_driver.c:250:50: warning: '%s' directive output may be truncated writing up to 63 bytes into a region of size between 51 and 61 [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wformat-truncation�-Wformat-truncation=�]8;;�]
  250 |                         snprintf (buf2, 64, "P%d_%s", (int)chn, buf); // needed to avoid duplicate names
      |                                                  ^~             ~~~
In function 'snprintf',
    inlined from 'ffado_driver_attach' at ffado_driver.c:250:4:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/bits/stdio2.h:68:10: note: '__builtin___snprintf_chk' output between 4 and 77 bytes into a destination of size 64
   68 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   69 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   70 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
ffado_driver.c: In function 'ffado_driver_attach':
ffado_driver.c:231:50: warning: '%s' directive output may be truncated writing up to 63 bytes into a region of size between 51 and 61 [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wformat-truncation�-Wformat-truncation=�]8;;�]
  231 |                         snprintf (buf2, 64, "P%d_%s", (int)chn, buf); // needed to avoid duplicate names
      |                                                  ^~             ~~~
In function 'snprintf',
    inlined from 'ffado_driver_attach' at ffado_driver.c:231:4:
/nix/store/kj8hbqx4ds9qm9mq7hyikxyfwwg13kzj-glibc-2.40-36-dev/include/bits/stdio2.h:68:10: note: '__builtin___snprintf_chk' output between 4 and 77 bytes into a destination of size 64
   68 |   return __builtin___snprintf_chk (__s, __n, __USE_FORTIFY_LEVEL - 1,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   69 |                                    __glibc_objsize (__s), __fmt,
      |                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
   70 |                                    __va_arg_pack ());
      |                                    ~~~~~~~~~~~~~~~~~
  CCLD     jack_firewire.la
make[3]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/drivers/firewire'
Making all in netjack
make[3]: Entering directory '/build/jack-audio-connection-kit-0.125.0/drivers/netjack'
  CC       libnetjack_packet_la-netjack_packet.lo
  CCLD     libnetjack_packet.la
ar: `u' modifier ignored since `D' is the default (see `U')
  CC       jack_net_la-net_driver.lo
  CC       jack_net_la-netjack_packet.lo
  CC       jack_net_la-netjack.lo
netjack.c: In function 'netjack_attach':
netjack.c:416:111: error: passing argument 1 of 'celt_decoder_create' makes integer from pointer without a cast [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wint-conversion�-Wint-conversion�]8;;�]
  416 |                         netj->capture_srcs = jack_slist_append (netj->capture_srcs, celt_decoder_create ( netj->celt_mode, 1, NULL ) );
      |                                                                                                           ~~~~^~~~~~~~~~~
      |                                                                                                               |
      |                                                                                                               CELTMode *
In file included from netjack.h:35,
                 from netjack.c:55:
/nix/store/lvrrkldia9sdzpyd6pr7v3zdfqzy3n8b-celt-0.11.3-dev/include/celt/celt.h:264:45: note: expected 'int' but argument is of type 'CELTMode *'
  264 | EXPORT CELTDecoder *celt_decoder_create(int sampling_rate, int channels, int *error);
      |                                         ~~~~^~~~~~~~~~~~~
netjack.c:463:109: error: passing argument 1 of 'celt_encoder_create' makes integer from pointer without a cast [�]8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wint-conversion�-Wint-conversion�]8;;�]
  463 |                         netj->playback_srcs = jack_slist_append (netj->playback_srcs, celt_encoder_create ( celt_mode, 1, NULL ) );
      |                                                                                                             ^~~~~~~~~
      |                                                                                                             |
      |                                                                                                             CELTMode *
/nix/store/lvrrkldia9sdzpyd6pr7v3zdfqzy3n8b-celt-0.11.3-dev/include/celt/celt.h:184:45: note: expected 'int' but argument is of type 'CELTMode *'
  184 | EXPORT CELTEncoder *celt_encoder_create(int sampling_rate, int channels, int *error);
      |                                         ~~~~^~~~~~~~~~~~~
make[3]: *** [Makefile:515: jack_net_la-netjack.lo] Error 1
make[3]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/drivers/netjack'
make[2]: *** [Makefile:398: all-recursive] Error 1
make[2]: Leaving directory '/build/jack-audio-connection-kit-0.125.0/drivers'
make[1]: *** [Makefile:537: all-recursive] Error 1
make[1]: Leaving directory '/build/jack-audio-connection-kit-0.125.0'
make: *** [Makefile:422: all] Error 2
```

</details>

## Additional context

@woffs @peti This also breaks shotcut.

```console
❯ git bisect start de1864217bfa9b5845f465e771e0ecb48b30e02d 4989a246d7a390a859852baddb1013f825435cee
[…]
❯ git bisect run nix-build -A jack1 --no-out-link
[gave up after a few hours]
```

## Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.5, NixOS, 25.05 (Warbler), 25.05.20241217.4989a24`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.11`
 - channels(root): `""`
 - nixpkgs: `/nix/store/9bdl6wa2qck3q0f7l892plk5lwy3pvx1-source`

## Notify maintainers

@infinisil @Artturin @Lurkki14

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/misc/jackaudio/jack1.nix
+++ b/pkgs/misc/jackaudio/jack1.nix
@@ -9,7 +9,7 @@
   db ? null,
   libuuid ? null,
   libffado ? null,
-  celt ? null,
+  celt_0_7 ? null,
 
   testers,
 }:
@@ -22,15 +22,15 @@ let
   optDb = shouldUsePkg db;
   optLibuuid = shouldUsePkg libuuid;
   optLibffado = shouldUsePkg libffado;
-  optCelt = shouldUsePkg celt;
+  optCelt = shouldUsePkg celt_0_7;
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "jack1";
-  version = "0.125.0";
+  version = "0.126.0";
 
   src = fetchurl {
-    url = "https://jackaudio.org/downloads/jack-audio-connection-kit-${finalAttrs.version}.tar.gz";
-    sha256 = "0i6l25dmfk2ji2lrakqq9icnwjxklgcjzzk65dmsff91z2zva5rm";
+    url = "https://github.com/jackaudio/jack1/releases/download/${finalAttrs.version}/jack1-${finalAttrs.version}.tar.gz";
+    hash = "sha256-eykOnce5JirDKNQe74DBBTyXAT76y++jBHfLmypUReo=";
   };
 
   configureFlags = [

